src/plugins/data/config.ts

```diff
@@ -51,6 +51,7 @@ export const searchConfigSchema = schema.object({
     waitForCompletion: schema.duration({ defaultValue: '100ms' }),
     keepAlive: schema.duration({ defaultValue: '1m' }),
     batchedReduceSize: schema.number({ defaultValue: 64 }),
+    pollInterval: schema.maybe(schema.number({ min: 1000 })),
```

Comment:
Nit: Can you provide comments for each of these config options? I know it's not completely related to this PR, but I think it'd be helpful, especially since this is the first option that isn't directly tied to an Elasticsearch parameter.